src/Makefile.test.include

```diff
@@ -85,7 +85,6 @@ BITCOIN_TESTS =\
   test/coins_tests.cpp \
   test/coinstatsindex_tests.cpp \
   test/compilerbug_tests.cpp \
-  test/compress_tests.cpp \
```

Comment:
In 1d376496fe9ac7110d0fac773ae5b5d9ef32aa59:

Did you intend to delete this?

Reply:
Was added back in a future commit but I went back and removed the change entirely.